### ISSUE TITLE
OpenAI api

### ISSUE THREAD

**Issue by Gourav1910** [exoascension/vault-chat#37]
Hi I really like your plugin idea but I can't use it because I don't have credit card and open ai is only supporting credit card. Is there any other way to get the api key?

**Comment by danglingptr0x0** [comment]
> Hi I really like your plugin idea but I can't use it because I don't have credit card and open ai is only supporting credit card. Is there any other way to get the api key?

No, there isn't. The plugin is currently made to work with the OpenAI API only, so you need their API key. And without having a payment-enabled account, you cannot get an API key.

**Comment by mdopp** [comment]
Wouldn't localai be possible? It's basically the same API.

https://localai.io/
